tests/backend/cores/abc_test_dbcore.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+# Import standard library
+import abc
+
+# Import modules
+import pytest
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.schema import Table
+
+
+class ABCTestDBCore(abc.ABC):
```

Comment:
Hmmm, I thought that anyone who subclasses `ABCTestDBCore` should always implement the fixtures. I guess I can remove it na lang!

Reply:
It's ok for APIs since it serves as a contract for usage but it's not so required in tests since it'll error out with the `NotImplementedError` exception anyways. Fixtures itself is already quite magical. This adds more learning overhead to contributors who don't know what ABC is.